src/core/jsproxy.c

```diff
@@ -719,10 +722,10 @@ JsProxy_toPy(PyObject* self,
   }
   PyObject* result =
     js2python_convert(GET_JSREF(self), depth, default_converter_js);
-  if (default_converter_js != NULL) {
+  if (pyproxy_Check(default_converter_js)) {
```

Comment:
BTW, unrelated to this PR, why is this `pyproxy_Check` function named in a strange mixed-case...?

Reply:
I think it should ideally be `PyProxy_Check`. Though it is confusing that a PyProxy isn't a Python type. I am trying to follow Python naming conventions they have eg PyType_Check and a bunch of other functions named similarly. But my naming is not all that consistent in general...